test/integration/targets/win_scheduled_task/tasks/triggers.yml

```diff
@@ -288,6 +288,84 @@
     that:
     - trigger_monthlydow_again is not changed
 
+- name: create trigger repetition (check mode)
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition_check
+  check_mode: yes
+
+- name: get result of create trigger repetition (check mode)
+  win_scheduled_task_stat:
+    path: \
+    name: '{{test_scheduled_task_name}}'
+  register: trigger_repetition_result_check
+
+- name: assert results of create trigger repetition (check mode)
+  assert:
+    that:
+    - trigger_repetition_check is changed
+    - trigger_repetition_result_check.task_exists == False
+
+- name: create trigger repetition
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition
+
+- name: get result of create trigger repetition
+  win_scheduled_task_stat:
+    path: \
+    name: '{{test_scheduled_task_name}}'
+  register: trigger_repetition_result
+
+- name: assert results of create trigger repetition
+  assert:
+    that:
+    - trigger_repetition is changed
+    - trigger_repetition_result.task_exists == True
+    - trigger_repetition_result.triggers|count == 1
+    - trigger_repetition_result.triggers[0].type == "TASK_TRIGGER_REGISTRATION"
+    - trigger_repetition_result.triggers[0].enabled == True
+    - trigger_repetition_result.triggers[0].repetition.interval == "PT1M"
+    - trigger_repetition_result.triggers[0].repetition.duration == "PT5M"
+    - trigger_repetition_result.triggers[0].repetition.stop_at_duration_end == True
+
+- name: create trigger repetition (idempotent)
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition_again
+
+- name: assert results of create trigger repetition (idempotent)
+  assert:
+    that:
+    - trigger_repetition_again is not changed
+
```

Comment:
I've only got one more request, are you able to create 2 more tests;

* modify trigger repetition (check mode)
* modify trigger repetition

This would modify the existing task but change one or two of the repetition values to something else and test accordingly. The only why this would be good to have is that this is the first object that is defined by a com object and having a test for this scenario would be great to have.